comp/venue/network.yaml

```diff
@@ -9,10 +9,22 @@ milestone: $SRYYYY Competition
 area-owner: event-logistics
 
 description: >-
-    This also requires that we have a pretty significant internet connection
-    given the number of people that are coming and our video streaming
-    requirement.
+  Internet access is required for several aspects of the competition.
+   - Allowing teams to access the documentation and Discord
+   - Allowing the livestream to be broadcast to YouTube
+   - Allowing publishing of scores to the website
 
-    If possible this should be mostly configured in a corner of the venue
-    as early as possible during setup and then, once working, deployed
-    to the proper locations.
+  For much of the production network, we use a wired network, connected to
+  the venue's internet connection. For the team pits, we use a WiFi network
+  that, for the last few years, has been provided by the venue (Eduroam).
```

Comment:
```suggestion
  that, for the last few years, has been provided by the venue.
```

Reply:
This is noting how we've previouslt done it, being explicit means we don't waste time having to hunt out the details of what guest accounts we previously needed.